sys/evtimer/evtimer.c

```diff
@@ -153,6 +153,25 @@ void evtimer_del(evtimer_t *evtimer, evtimer_event_t *event)
     irq_restore(state);
 }
 
+uint32_t evtimer_get_offset(evtimer_t *evtimer, evtimer_event_t *event)
+{
+    uint32_t offset = 0;
+    unsigned state = irq_disable();
+    if (evtimer->events) {
+        evtimer_event_t *list = evtimer->events;
+        /* Get current offset to first event */
+        offset = _get_offset(&evtimer->timer);
+        while ((list = list->next)) {
```

Comment:
Doesn't this skip the first event now?

Reply:
The first event is supposed to be skipped as the current offset of the first event is retrieved at L163

Reply:
Got it!